sacc2/sacc.py

```diff
@@ -0,0 +1,1026 @@
+import numpy as np
+import copy
+import os
+import yaml
+import warnings
+import pickle
+
+from collections import OrderedDict
+from astropy.io import fits
+from astropy.table import Table, Column
+
+from .tracers import BaseTracer
+from .windows import BaseWindow
+from .covariance import BaseCovariance
+from .utils import unique_list
+
+
+# add more as we develop them
+allowed_types = [
+    "shear_xi_plus",
+    "shear_xi_minus",
```

Comment:
In all generality, these things have imaginary components caused by parity-breaking correlations (i.e. EB power spectra). You may want to include those, in the same way you're including `ggl_gamma_x`, which is the imaginary part of the ggl correlation function.

Reply:
added